inc/saiswitch.h

```diff
@@ -471,6 +487,37 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_DEFAULT_VIRTUAL_ROUTER_ID,
 
+    /** 
+     * @brief Switch/Global bind point for ingress ACL object
+     *
+     * Bind (or unbind) an ingress acl table or acl group globally. Enable/Update 
+     * ingress ACL table or ACL group filtering by assigning the list of valid 
+     * object id . Disable ingress filtering by assigning SAI_NULL_OBJECT_ID 
+     * in the attribute value. 
+     *
+     * @type sai_object_id_t
+     * @objects SAI_OBJECT_TYPE_ACL_TABLE, SAI_OBJECT_TYPE_ACL_TABLE_GROUP
+     * @flags CREATE_AND_SET
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_SWITCH_ATTR_DEFAULT_INGRESS_ACL,
```

Comment:
SAI_SWITCH_ATTR_DEFAULT_INGRESS_ACL --> SAI_SWITCH_ATTR_INGRESS_ACL can make it similar to other objects